config.json

```diff
@@ -3,6 +3,7 @@
     { "tag": "6", "from": "node:6" },
     { "tag": "6-npm6.1", "from": "node:6", "extraInstructions": ["RUN npm install npm@6.1.x", "RUN rm -rf /usr/local/lib/node_modules/npm", "RUN mv node_modules/npm /usr/local/lib/node_modules/npm"] },
     { "tag": "8", "from": "node:8" },
-    { "tag": "10", "from": "node:10" }
+    { "tag": "10", "from": "node:10" },
+    { "tag": "12", "from": "node:12" }
```

Comment:
I think that would work, I just didn't know if that was something we wanted to do. Obviously, minor/patch versions come out more frequently than major ones, so we'd have to add new ones more often in order to stay up-to-date. I don't know how annoying it would be for people if I made a new "add 12.x.y" image every time they released a new one.

Reply:
I don't think we'd have to do every one, just the ones we felt were necessary. It's not like the major version ones get updated all the time anyways.